File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int j = 0;
        for(int i = 1;i<nums.size();i++) {
            if(nums[j] != nums[i]) {
                j+=1 ;
                nums[j] = nums[i];
            }
        }
        return j+1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let j = 0;
    let count = 0;
    for(let i = 1;i<nums.length;i++) {
        if(nums[j]!=nums[i]) {
            nums[j+1] = nums[i];
            count++
            j+=1
        }
    }
    return j+1;
};